src/translations/translations.fr.js

```diff
@@ -226,6 +226,65 @@ export const translations = {
   //#region cc-doc-list
   'cc-doc-list.error': `Une erreur est survenue pendant le chargement de la documentation`,
   //#endregion
+  //#region cc-domain-management
+  'cc-domain-management.certif.automated': () => sanitize`Que vous utilisiez <code>cleverapps.io</code> ou vos propres noms de domaine avec les applications hébergées par Clever Cloud, un certificat Let's Encrypt est automatiquement généré et renouvelé pour l'accès HTTPS/TLS. Vous n'avez rien à faire, tout est automatisé. Pour les cas spécifiques, reportez vous à la page <a href="https://developers.clever-cloud.com/doc/administrate/ssl/"><span lang="en">Installing TLS Certificates</span> (en Anglais)</a>.`,
```

Comment:
question: I never realized we capitalized `Anglais` in french, I guess it's a rule?